docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: '3.8'
+services:
+  hashtopolis:
+    container_name: hashtopolis
+    build: .
+    restart: always
+    volumes:
+      ##
+      # !!! CAREFUL, importing from here will delete whatever file is in this folder !!!
+      - ./src/import/:/var/www/html/import/:Z
+      ##
+      - ./src/files/:/var/www/html/files/:Z
+      - ./data/conf.php:/var/www/html/inc/conf.php:Z
+    depends_on:
+      - db
+    ports:
+      - 8080:80
+    cap_drop:
+      - ALL
+    cap_add:
+      # used to change files since non-priv user
+      - CHOWN
+      # used for the gosu binary
+      - CAP_SETGID
+      - CAP_SETUID
+  db:
+    container_name: db
+    image: mysql:5.7
+    restart: always
+    volumes:
+      - ./db:/var/lib/mysql:Z
```

Comment:
Move this to the data folder.

Reply:
[done](https://github.com/elreydetoda/hashtopolis-server/commit/8a7dba6b14fe2693cffe0b86c11561ee9ff7a796)